src/guides/v2.3/frontend-dev-guide/css-topics/css-preprocess.md

```diff
@@ -101,7 +101,10 @@ The tool pre-processes (including compilation) and publishes the static view fil
 {:.bs-callout-info}
 Manual static content deployment is not required in "default" and "developer" modes. If you still want to deploy in these modes, use the -f option: `bin/magento setup:static-content:deploy -f`. Read more about the command in the [Deploy static view files]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-static-view.html) section.
 
-All errors occurring during `.less` files compilation are handled by the [`oyejorge/less.php`](https://github.com/oyejorge/less.php) third party library.
+All errors occurring during `.less` files compilation are handled by the [`wikimedia/less.php`]([LESS PHP library]) third party library.
+
+{:.bs-callout-info}
+Since Magento Open Source 2.3.4 package "oyegorge/less.php" has already replaced by "wikimedia/less.php"
```

Comment:
Can you please say why Open Source has been specifically mentioned as that implies Commerce version is not affected?
```suggestion
Since Magento Open Source 2.3.4, `oyegorge/less.php` has been replaced by `wikimedia/less.php`
```

Reply:
@mrtuvn: I would just drop "Open Source" to be honest, as it applies to both Open Source and Commerce...

Just say `Magento 2.3.3`

Btw: note that the package made it's introduction in 2.3.3 and not in 2.3.4:
- 2.3.2: https://github.com/magento/magento2/blob/2.3.2/composer.json#L43
- 2.3.3: https://github.com/magento/magento2/blob/2.3.3/composer.json#L43

By https://github.com/magento/magento2/commit/9a4332cb18945772c1458d53cd1a9c191c3f7a10

Reply:
Updated as your suggestions